.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+{
+    "python.linting.pylintEnabled": false,
+    "python.linting.flake8Enabled": true,
+    "python.linting.enabled": true,
+    "python.pythonPath": "C:\\Users\\ajtoo\\.virtualenvs\\JoeBoganBotcast-sIpSHciI\\Scripts\\python.exe"
```

Comment:
checking this in isn't going to work well. Maybe a env-var or something? I've been using Pycharm